apps/aecore/include/trees.hrl

```diff
@@ -1,13 +1,12 @@
--include("common.hrl").
-
 -record(trees, {
           accounts}).
 -type(trees() :: #trees{}).
 
 %% Placeholder to define state Merkle trees
 
 -record(account, {
-          pubkey = <<>>  :: pubkey(),
```

Comment:
I would suggest keeping type pubkey as it is a type used over several places of the code and clarifies link back to the account.